src/demo/app/global-css/reset-demo.component.ts

```diff
@@ -0,0 +1,87 @@
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'reset-demo',
+  template: `
+    <h2>CSS reset</h2>
+    <p class="desc">A style sheet designed to strip initial styles from browsers to help keep designs consistent</p>
+    <ul class="doc">
+      <li><code>display: block</code> on HTML5 block level elements</li>
+      <li>Default border, margin, and padding of zero</li>
+      <li><code>vertical-align: baseline</code> is the consistent default</li>
+      <li><code>font-size: 100%</code> and <code>font-weight: 400</code> defaults</li>
+      <li><code>background: 0 0</code> for consistent default background properties of no background</li>
+      <li><code>box-sizing: border-box</code> sizing so the width and height properties (and min/max properties)
+        includes content, padding and border, but not the margin</li>
+      <li><code>max-width: 100%</code> for responsive <code>embed</code>, <code>img</code>, <code>object</code></li>
+      <li><code>list-style: none</code> on <code>ul</code> elements</li>
+      <li>
+        Remove quotes from <code>blockquote</code> and <code>q</code>
+        <aside class="example">Examples: <q>This is an inline <code>q</code>.</q>
+          <blockquote>
+            Hello, I am a blockquote. My browser provided margin has been zeroed due to this reset.
+          </blockquote>
+        </aside>
+      </li>
+      <li>
+        <code>text-decoration: line-through</code> on <code>del</code> 
+        <del>This text has been deleted from the document.</del>
+      </li>
+      <li>
+        A default <dfn title="dfn example">dotted underline border</dfn> and <abbr title="abbr example">help cursor</abbr> for 
+        <code>abbr</code> and <code>dfn</code> elements with <code>title</code> help text
+        <aside class="example">
+          <code>
+            &lt;abbr title="help text"&gt;<abbr 
+              title="abbr should include title attribute for help text">How to</abbr>&lt;/abbr&gt;
+          </code>
+        </aside>
+      </li>
+      <li>
+        Zero padding, 1em margin, and 1px height and <code>@grey-silver</code> border-top for <code>hr</code> elements
+      </li>
+      <li>
+        Allows unbreakable words to be broken and white space preserved on <code>pre</code> elements
+        <aside class="example">
+          <pre>
+            EXAMPLE:
```

Comment:
Kind of surprised at myself even that this styling is as consistent as it is because it sort of developed over time as I'd look at it again and say "eh, not quite what I want"